cdk/domino_cdk/config/s3.py

```diff
@@ -1,48 +1,86 @@
 from dataclasses import dataclass
+from os import getenv
 from typing import Dict, Optional
 
-from domino_cdk.config.util import from_loader
+from domino_cdk.config.util import check_leavins, from_loader
 
 
 @dataclass
 class S3:
     """
     Map of buckets to create and provide IAM access from the EKS cluster.
 
-    The monitoring bucket is optionally created to store various operational logs: VPC flow logs, S3 and ELB access logs.
+    "blobs", "logs", "backups" and "registry" buckets are MANDATORY.
+
+    The "monitoring" bucket is optionally created to store various operational logs: VPC flow logs, S3 and ELB access logs.
 
     Bucket parameters:
+    name: "my-bucket-name" - Optional name to override default bucketname of deployname-bucket-type
```

Comment:
Is this something we really want to support configuring or just an easy toss-in?

Reply:
We can configure buckets in the deployment config. So here is a must.

I'm more concerned about 2 sources of truth: deployment config and provisioning config (cdk config). There is stuff to be configured in both like bucket names, k8s namespaces.

It would be good to have one config instead of multiple.

Reply:
Since s3 buckets are globally namespaced (and occasionally s3 is slow to release names when iterating), I thought it necessary to have a safety valve with the bucket names. I _don't_ want people configuring it, which is why I added the concept of the hidden option for it.

Perhaps documenting it there is still too visible? I kind of fear that _we'll_ forget we can even do it in the head of the moment, though. :)